server/service/src/sync/mod.rs

```diff
@@ -47,6 +47,18 @@ pub(crate) struct ActiveStoresOnSite {
 pub(crate) fn get_sync_push_changelogs_filter(
     connection: &StorageConnection,
 ) -> Result<Option<ChangelogFilter>, GetActiveStoresOnSiteError> {
+    if CentralServerConfig::is_central_server() {
+        // If this is a central server, we want to send everything that that wasn't from legacy site
+        let msupply_central_server_id = KeyValueStoreRepository::new(connection)
+            .get_i32(repository::KeyType::SettingsSyncCentralServerSiteId)
+            .map_err(|_| GetActiveStoresOnSiteError::CentralSiteIdNotSet)?
```

Comment:
This is note GetActiveStoreOnSiteError, we might need another error type for get_sync_push_changelogs_filter rather then re-using get active store on site error

Reply:
Also for this one in particular there is repository error variant, but the one below yep, something else